rules/windows/process_creation/Conti_esentutl.yaml

```diff
@@ -0,0 +1,32 @@
+title: Detection of esentutl aka Extensible Storage Engine Utilities to gather credentials
+id: 7df1713a-1a5b-4a4b-a071-dc83b144a101
+status: experimental
+author: sam0x90
+date: 2021/08/06
+description: Conti recommendation to its affiliates to use esentult to access NTDS dumped file. Trickbot also uses this utilities to get MSEdge info via its module "pwgrab"
+references:
+    - https://twitter.com/vxunderground/status/1423336151860002816
+	- https://attack.mitre.org/software/S0404/
+    - https://thedfirreport.com/2021/08/01/bazarcall-to-conti-ransomware-via-trickbot-and-cobalt-strike/
+tags:
+    - attack.credential_access
+    - attack.t1003
+    - attack.t1003.003
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains:
+            - 'esentutl'
+    selection_password:
+        CommandLine|contains: ' /p'
+    condition: all of them

```

Comment:
Other way:
```yaml
   selection:
        CommandLine|contains|all:
            - 'esentutl'
            - ' /p'
    condition: selection
```
Fisrt way : `(event.code:"1" AND winlog.channel:"Microsoft\-Windows\-Sysmon\/Operational" AND process.command_line.keyword:*esentutl* AND process.command_line.keyword:*\ \/p*)`
Other way : alfer `(event.code:"1" AND winlog.channel:"Microsoft\-Windows\-Sysmon\/Operational" AND process.command_line.keyword:*esentutl* AND process.command_line.keyword:*\ \/p*)`
Same result